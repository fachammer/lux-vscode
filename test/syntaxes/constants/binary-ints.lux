0b01
0b10