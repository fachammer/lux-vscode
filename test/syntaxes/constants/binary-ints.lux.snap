>0b01
#^^^^ source.lux constant.numeric.int.binary.lux
>0b10
#^^^^ source.lux constant.numeric.int.binary.lux